### ISSUE TITLE
关于训练voc_dataset结果

### ISSUE THREAD

**Issue by snow-wind-001** [YunYang1994/tensorflow-yolov3#161]
训练了30个epoch,mAP能达到0.72
![mAP](https://user-images.githubusercontent.com/33013904/58149350-2be65f00-7c95-11e9-9d07-b4612df5526a.png)

**Comment by yangyangtiaoguo** [comment]
......我怎么就50轮才六十多。。。。。。

**Comment by yangyangtiaoguo** [comment]
batch size 16.就改了这个，单GPU训练

**Comment by yangyangtiaoguo** [comment]
你的loss降到多少了？

**Comment by change113** [comment]
> 时代

你的30个epoch是什么意思？在作者代码中是如何设置30个epoch的？谢谢

**Comment by snow-wind-001** [comment]
试了一些参数，mAP达到0.88
![mAP](https://user-images.githubusercontent.com/33013904/58227054-dd4fc800-7d5b-11e9-85aa-67854292fbe0.png)

**Comment by snow-wind-001** [comment]
> > 时代
> 
> 你的30个epoch是什么意思？在作者代码中是如何设置30个epoch的？谢谢

是的，在config.py中改一些参数

**Comment by snow-wind-001** [comment]
> 你的loss降到多少了？



> 你的loss降到多少了？
test loss = 7左右

**Comment by snow-wind-001** [comment]
> > 时代
> 
> 你的30个epoch是什么意思？在作者代码中是如何设置30个epoch的？谢谢

在config.py改的

**Comment by change113** [comment]
> > > 时代
> > 
> > 
> > 你的30个时代是什么意思？在作者代码中是如何设置30个划时代的？谢谢
> 
> 在config.py改的

你是用的作者最新的代码吗？

**Comment by yangyangtiaoguo** [comment]
> 时代
> 
> 你的30个epoch是什么意思？在作者代码中是如何设置30个epoch的？谢谢

你这感觉啥都不懂呀。。。。。。30就是训练30轮，每个人机器不一样，你改了也没用。。。

**Comment by yangyangtiaoguo** [comment]
> 试了一些参数，mAP达到0.88

请问，可以共享下你的参数设置嘛？

**Comment by change113** [comment]
> > 时代
> > 你的30个epoch是什么意思？在作者代码中是如何设置30个epoch的？谢谢
> 
> 你这感觉啥都不懂呀。。。。。。30就是训练30轮，每个人机器不一样，你改了也没用。。。
刚入门不久。可以留个联系方式吗。534739458我QQ。方便加一下吗？谢谢

**Comment by change113** [comment]
> > 时代
> > 你的30个epoch是什么意思？在作者代码中是如何设置30个epoch的？谢谢
> 
> 你这感觉啥都不懂呀。。。。。。30就是训练30轮，每个人机器不一样，你改了也没用。。。

刚入门不久，方便加我一下吗？534739457我的QQ。谢谢

**Comment by yyaqi** [comment]
您好，可以共享一下您的参数设置吗？

**Comment by change113** [comment]
你好：
BatchNorm / moving_variance 
['yolov3 / yolo-v3 / Conv_6 / weights'，TensorShape（[Dimension（1），Dimension（1），Dimension（1024），Dimension（255）]）] 
['conv_lbbox / weight'，TensorShape（ [Dimension（1），Dimension（1），Dimension（1024），Dimension（75）]）] 
Traceback（最近一次调用最后一次）：
文件“convert_weight.py”，第66行，in 
raise RuntimeError 
RuntimeError 

这个“长度不等的问题”，你有遇到过吗？怎么解决的？谢谢

**Comment by yulei1234** [comment]
> 试了一些参数，mAP达到0.88
> ![mAP](https://user-images.githubusercontent.com/33013904/58227054-dd4fc800-7d5b-11e9-85aa-67854292fbe0.png)

请问一下，你是加载了coco的权重吗，有哪些参数可以尝试着改动呢，试了两天也没到你那个精度，谢谢

**Comment by snow-wind-001** [comment]
加载了预权重，调整了epoch大小，训练分多段进行，观察最终test的损失，实时进行调整。

**Comment by ye-ye** [comment]
@snow-wind-001  您好。我最近在用yolov3训练voc，用的原代码效果并不好，可以加您的联系方式交流一下吗，vx:ye1337999794

**Comment by yunyao01** [comment]
> 文件“convert_weight.py”，第66行，in
我也遇到同样的问题，请问你的问题解决了吗？

**Comment by tensorflowt** [comment]
> 试了一些参数，mAP达到0.88
> ![mAP](https://user-images.githubusercontent.com/33013904/58227054-dd4fc800-7d5b-11e9-85aa-67854292fbe0.png)

你好！请问你的train_first_stage_epochs、train_second_stage_epochs分别设定为多少？

**Comment by beibeiZ** [comment]
> 训练了30个epoch,mAP能达到0.72
> ![mAP](https://user-images.githubusercontent.com/33013904/58149350-2be65f00-7c95-11e9-9d07-b4612df5526a.png)

![image](https://user-images.githubusercontent.com/47970256/63479124-698a9c00-c4bf-11e9-9a6c-d9dde3009461.png)
我的一个enpoch四个多小时正常吗

**Comment by wuranran** [comment]
训练test___loss为8.02运行evaluate以后有检测结果，但是计算的ap为0不知道怎么回事？请问这是正常得嘛，

**Comment by wuranran** [comment]
训练test___loss为8.02运行evaluate以后有检测结果，但是计算的ap为0不知道怎么回事？请问这是正常得嘛，

**Comment by navy63** [comment]
> > 训练了30个epoch,mAP能达到0.72
> > ![mAP](https://user-images.githubusercontent.com/33013904/58149350-2be65f00-7c95-11e9-9d07-b4612df5526a.png)
> 
> ![image](https://user-images.githubusercontent.com/47970256/63479124-698a9c00-c4bf-11e9-9a6c-d9dde3009461.png)
> 我的一个enpoch四个多小时正常吗

应该不正常，我一个epoch大概15分钟不到，GTX2080TI

**Comment by navy63** [comment]
> ......我怎么就50轮才六十多。。。。。。

yolov3_test_loss=9.7718.ckpt-50  mAP=0.70
请问知道什么原因么，训练50周期也才70

**Comment by navy63** [comment]
> 试了一些参数，mAP达到0.88
> ![mAP](https://user-images.githubusercontent.com/33013904/58227054-dd4fc800-7d5b-11e9-85aa-67854292fbe0.png)

能告知调了那些参数么，跪求

**Comment by 16534165** [comment]
> 你好：
> BatchNorm / moving_variance
> ['yolov3 / yolo-v3 / Conv_6 / weights'，TensorShape（[Dimension（1），Dimension（1），Dimension（1024），Dimension（255）]）]
> ['conv_lbbox / weight'，TensorShape（ [Dimension（1），Dimension（1），Dimension（1024），Dimension（75）]）]
> Traceback（最近一次调用最后一次）：
> 文件“convert_weight.py”，第66行，in
> raise RuntimeError
> RuntimeError
> 
> 这个“长度不等的问题”，你有遇到过吗？怎么解决的？谢谢


你解决了吗兄弟

**Comment by hanxu1202** [comment]
> 试了一些参数，mAP达到0.88
> ![mAP](https://user-images.githubusercontent.com/33013904/58227054-dd4fc800-7d5b-11e9-85aa-67854292fbe0.png)

您好，请问是否方便分享下跑VOC数据集的参数，batch size、学习率、输入尺寸等

**Comment by cxjaicj** [comment]
> 试了一些参数，mAP达到0.88
> ![mAP](https://user-images.githubusercontent.com/33013904/58227054-dd4fc800-7d5b-11e9-85aa-67854292fbe0.png)
请问能不能分享一下VOC的权重？万分感谢 QQ1552006123

**Comment by cxjaicj** [comment]
> > 试了一些参数，mAP达到0.88
> > ![mAP](https://user-images.githubusercontent.com/33013904/58227054-dd4fc800-7d5b-11e9-85aa-67854292fbe0.png)
> 
> 请问一下，你是加载了coco的权重吗，有哪些参数可以尝试着改动呢，试了两天也没到你那个精度，谢谢

请问能不能分享一下你的VOC训练权重，我一直训练不出好结果？QQ1552006123

**Comment by cxjaicj** [comment]
> 训练了30个epoch,mAP能达到0.72
> ![mAP](https://user-images.githubusercontent.com/33013904/58149350-2be65f00-7c95-11e9-9d07-b4612df5526a.png)

请问能不能分享一下你的VOC权重 ?QQ1552006123  谢谢
